plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -25,6 +25,10 @@ class ServiceModals extends React.Component {
       pendingActions
     } = this.props;
 
+    if (modalProps.service == null) {
```

Comment:
@weblancaster are you sure this is the correct place to fix? Seems odd fixing this issue by not rendering the Group modal

Reply:
Makes sense, let me take a look

Reply:
You were right @MatApple the issue goes deep in the rabbit hole.

Reply:
I agree with both of you, but as this is a blocker, let's get it merged and find a proper fix next print when there is more time. I added an issue so that we don't forget 👍 https://jira.mesosphere.com/browse/DCOS-20391